opm/autodiff/FlowMainEbos.hpp

```diff
@@ -707,8 +700,17 @@ namespace Opm
         Grid& grid()
         { return ebosSimulator_->gridManager().grid(); }
 
+        Problem& ebosProblem()
```

Comment:
ok, I tried to remove the non-const version. that produced an error because the non-const materialLawManager() method is needed (for geology updates during the simulation (?)) and this patch uses ebosProblem() for it. We could go back to the previous version for materialLawManager(), but I think using ebosProblem() is slightly more elegant...